homeassistant/components/mqtt/quality_scale.yaml

```diff
@@ -0,0 +1,131 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      Service actions are set up in `async_setup_entry` and are only set up if the MQTT broker was set up correctly.
```

Comment:
I think this is the point of action-setup is to set them up in `async_setup` to make sure they are not dependent on a correctly setup broker, to at least make the validation of services in automations work

Reply:
I'll make this a todo

Reply:
https://github.com/home-assistant/core/pull/131170

Reply:
Change was merged, state is set to `done`